.github/workflows/update-maintainers.yml

```diff
@@ -0,0 +1,245 @@
+name: Update maintainers list
+
+on:
+  push:
+    branches: ["main"]
+    paths: ["lib/maintainers.nix"]
+  schedule:
+    # Update every Monday at 9 AM UTC
+    - cron: "0 9 * * 1"
+  workflow_dispatch:
+    inputs:
+      create_pr:
+        description: "Create PR even if no changes"
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  update-maintainers:
+    runs-on: ubuntu-latest
+    # Creating PRs to update the generated maintainers file
```

Comment:
Should this comment be at the head of the file? Immediately before the job? It feels out of place here before the `permissions` block.

This _would_ be a good location to mention that we explicitly support `github.token` for easier testing on forks, though.

Reply:
It was to explain the permissions